tough-ssm/src/lib.rs

```diff
@@ -58,15 +58,17 @@ impl KeySource for SsmKeySource {
         key_id_hex: &str,
     ) -> std::result::Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
         let ssm_client = client::build_client(self.profile.as_deref())?;
-        let fut = ssm_client.put_parameter(rusoto_ssm::PutParameterRequest {
-            name: self.parameter_name.to_owned(),
-            description: Some(key_id_hex.to_owned()),
-            key_id: self.key_id.as_ref().cloned(),
-            overwrite: Some(true),
-            type_: Some("SecureString".to_owned()),
-            value: value.to_owned(),
-            ..rusoto_ssm::PutParameterRequest::default()
-        });
+
+        let fut = ssm_client
+            .put_parameter()
+            .name(self.parameter_name.to_owned())
+            .description(key_id_hex.to_owned())
+            .set_key_id(self.key_id.as_ref().cloned())
+            .overwrite(true)
+            .set_type(Some(aws_sdk_ssm::model::ParameterType::SecureString))
+            .value(value.to_owned())
+            .send();
+
         tokio::runtime::Runtime::new()
```

Comment:
As I mentioned elsewhere, we should consider spawning a thread before doing this to avoid runtime-in-runtime panic, but I'm don't know that we need to change existing cases of this in the current PR.